sql/core/src/main/scala/org/apache/spark/sql/internal/CatalogImpl.scala

```diff
@@ -138,7 +138,7 @@ class CatalogImpl(sparkSession: SparkSession) extends Catalog {
    */
   @throws[AnalysisException]("table does not exist")
   override def listColumns(tableName: String): Dataset[Column] = {
```

Comment:
The documentation in org.apache.spark.sql.catalog.Catalog:
```scala
 /**
  * Returns a list of columns for the given table in the current database.
  *
  * @since 2.0.0
  */
  @throws[AnalysisException]("table does not exist")
  def listColumns(tableName: String): Dataset[Column]
```

Should be updated...